lib/classes/popover.js

```diff
@@ -67,6 +67,17 @@ class PopOver extends ToolTip {
         }
     };
 
+    getTitle() {
+        let title = this.$config.title || '';
+        if (typeof title === 'function') {
+            title = title.call(this.$element).toString();
+        }
+        if (!title) {
```

Comment:
I think we need to chek here to see if title is also a DOM node (`typof title === 'object' && title.nodeType && title.innerHTML.trim())`), and if so, just return the title as a node

Reply:
This isnt work with current example. Function returns object, and then it stringified to `[object Object]`